From 432afcd9515781d6f9c347cebaad0974e65c13d5 Mon Sep 17 00:00:00 2001
From: Manjot Sidhu <manjot.techie@gmail.com>
Date: 3 Jun, 2018 6:15:19 PM

Switch to predefined stuff

diff --git a/src/tictactoe/TicTacToe.java b/src/tictactoe/TicTacToe.java
index 8e36113..53edb1c 100644
--- a/src/tictactoe/TicTacToe.java
+++ b/src/tictactoe/TicTacToe.java
@@ -18,13 +18,13 @@
     // Parameterss
     private int ticLength;
     // ticArray predefined
-    /* private int[][] ticArray = {{1,2,1,1,1,2},
+    private int[][] ticArray = {{1,2,1,1,1,2},
                                 {7,1,9,1,1,2},
                                 {1,8,9,1,1,2},
                                 {7,8,9,1,1,2},
                                 {7,8,9,1,1,2},
-                                {7,8,9,1,1,2}}; */
-    private int[][] ticArray;
+                                {7,8,9,1,1,2}};
+    //private int[][] ticArray;
     
     private int u1 = 1;
     private int u2 = 2;
@@ -40,11 +40,11 @@
     public void init() {
         System.out.print(MSS_GREEN + "Enter Length/Order: " + MSS_RESET);
 
-        this.ticLength = scan.nextInt();
+        //this.ticLength = scan.nextInt();
         // ticLength predefined
-        //this.ticLength = 6;
+        this.ticLength = 6;
 
-        ticArray = new int[ticLength][ticLength];
+        //ticArray = new int[ticLength][ticLength];
 
     }
 
